commons/mixins.py

```diff
@@ -1,8 +1,11 @@
+from django.http import QueryDict
+
+
 class BasePopulateDataMixin:
     def update_request(self, request):
-        request.data._mutable = True
+        if isinstance(request.data, QueryDict):
```

Comment:
It would be better if we check existence of `_mutable` attribute rather than check data's type. So we check what we use.
```suggestion
        if hasattr(request.data, "_mutable") and request.data._mutable is False:
```